### PR TITLE
UICLAIM-7 Setup new UI project `ui-claims`

### PR DESCRIPTION
**Pull request by usavkov-epam**
Initial project setup

### CODE REVIEW

File: .github/workflows/build-npm-release.yml [hunk 1]
```diff
@@ -0,0 +1,231 @@
+# This workflow will do a clean install of node dependencies, build the source code,
```
Comment (BogdanDenis) [inline .github/workflows/build-npm-release.yml]: Please use new shared UI workflow
Example ticket [UINOTES-151](https://folio-org.atlassian.net/browse/UINOTES-151)

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

**Comment by usavkov-epam** [comment]
@folio-org/folio-devops could you please check GA actions and assist with sonar?

File: .github/workflows/build-npm-release.yml (hunk 1 above)
Comment (usavkov-epam) [reply .github/workflows/build-npm-release.yml]: updated

**Review by BogdanDenis — APPROVED** [review]

**Comment by julianladisch** [comment]
> @folio-org/folio-devops could you please check GA actions and assist with sonar?

Done.

**Comment by usavkov-epam** [comment]
@folio-org/acquisitions-ui please take a look

**Review by alisher-epam — APPROVED** [review]

**Pull request merged**